src/main.zig

```diff
@@ -327,7 +327,7 @@ test "access list filled" {
 
     var buf: [128]u8 = undefined;
     const rlp = try std.fmt.hexToBytes(&buf, "f83af838f7940000000000000000000000000000000000001210e1a00000000000000000000000000000000000000000000000000000000000000203");
-    var out: StrippedTxn = StrippedTxn{ .access_list = &[0]AccessListItem{} };
+    var out: StrippedTxn = undefined; // StrippedTxn{ .access_list = &[0]AccessListItem{} };
```

Comment:
It no longer makes sense to partially allocate the structure, at least I can not think of a use case just now.

```suggestion
    var out: StrippedTxn = undefined;
```

Reply:
Cool.